### PR TITLE
Move the FC_BN_HSWISH_ACTIVATION pattern from fused to ignored

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Changes

Move the `FC_BN_HSWISH_ACTIVATION` pattern from fused patterns to ignored patterns.

### Reason for changes

This is an incorrect fused pattern because we cannot fuse batch normalization in `Unsqueeze` operation. The correct fused pattern is

Convolution + Bias + Activation + Unsqueeze + BatchNormalization + Squeeze

### Related tickets

Ref: 112991

### Tests

N/A

### CODE REVIEW

**Comment by andrey-churkin** [comment]
post_training_quantization build # 66

**Review by alexsu52 — APPROVED** [review]
LGTM

**Comment by andrey-churkin** [comment]
run pre-commit tests

**Comment by andrey-churkin** [comment]
run pylint pre-commit tests
run pytorch pre-commit tests
run openvino pre-commit tests
run onnx pre-commit tests
